### DISCUSSION TITLE
Support for Layouts?

### DISCUSSION THREAD

**Discussion by TechplexEngineer** [livebud/bud#98] [Q&A]
Does bud support layouts like sveltekit?
`__layout.svelte` https://kit.svelte.dev/docs/layouts

**Comment by matthewmueller** [comment]
Not yet! It's in the code though, just needs to be exposed and tested: https://github.com/livebud/bud/blob/acf06551fffad5286e028af32ba082cc2416630d/runtime/view/ssr/svelte.ts
